_posts/en/newsletters/2024-09-06-newsletter.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Bitcoin Optech Newsletter #319'
+permalink: /en/newsletters/2024/09/06/
+name: 2024-09-06-newsletter
+slug: 2024-09-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Stratum v2 extension for fee revenue sharing:** Filippo Merli
+  [posted][merli stratumfees] to Delving Bitcoin about an extension to
+  [Stratum v2][topic pooled mining] that will allow tracking the amount
+  of fees included in _shares_ when the shares contain transactions
+  selected by an individual miner.  This can be used to adjust the
+  amount paid to the miner by the pool, with miners selecting higher
+  feerate transactions being paid more.
+
+  Merli links to a [paper][merli paper] he co-authored that examines
+  some of the challenges of paying different miners different amounts
+  based on the transactions they select, and which suggests a scheme
```

Comment:
```suggestion
  based on the transactions they select. The paper suggests a scheme
```
Initially, I was going to suggest replacing "which" with "that" (which I think would be a slight improvement), but this is a pretty long and complex sentence, so maybe break it into two.